lib/engines/hcktest/tests.rb

```diff
@@ -301,16 +301,23 @@ def collect_memory_dump(machine, l_tmp_path)
       true
     end
 
+    def download_memory_dumps(l_tmp_path)
+      collected_client = download_memory_dump(@client.name, "#{l_tmp_path}/#{@client.name}_#{current_timestamp}")
```

Comment:
We only download dumps here; please rename `collected_client` to `downloaded_client`. Do the same for `collected_support` too.